### DISCUSSION TITLE
Website not working without VPN in Nigeria

### DISCUSSION THREAD

**Discussion by JolomiTee** [igordanchenko/yet-another-react-lightbox#276] [General]
### Describe the bug

the website for the yet-another-react-lightbox found [here](https://yet-another-react-lightbox.com/documentation) does not open return the webpage except with VPNs

### Expected behavior

It should redirect to the webpage

### How to reproduce

I navigated to [the documentation site](https://yet-another-react-lightbox.com/documentation) without a VPN connection. I am located in Nigeria, West Africa

**Comment by igordanchenko** [comment]
It sounds like an issue specific to your network / ISP. 

Please feel free to add some additional details if you require further assistance.
- `curl -svo /dev/null https://yet-another-react-lightbox.com/documentation`
- `ping yet-another-react-lightbox.com`
- `dig yet-another-react-lightbox.com`
- `dig @8.8.8.8 yet-another-react-lightbox.com`
